src/lib/components/events/event-item.svelte

```diff
@@ -39,17 +38,20 @@
 <div class="event-box" style:--highlights={`var(--acm-${info.acmPath.slug}-rgb)`}>
   <!-- Workaround for the top panel covering the event card's anchor. -->
   <div class="anchor" id={info.slug} bind:this={anchor} />
+
   <details class="event-card" bind:this={details}>
     <summary class="event-body">
       <div class="event-name">
         <h2 class="headers">
           {info.title}
         </h2>
 
-        <p class="event-location">
-          {info.location === 'Discord' || info.location === 'Zoom'
-            ? `Hosted on ${info.location}`
-            : info.location}
+        <p class="event-location brand-med acm-gray">
```

Comment:
Is discord what gets inputted to indicate default? What happens with the case where it actually is hosted on discord?

Reply:
i clicked on the wrong line i meant to comment on the next one

Reply:
Discord is the default. Janky work around would be to set the location on GCal for an event to be 'Discord' if we are hosting on discord, but i'm willing to see if i can add something to make an exception